_includes/site-navigation/sidenav-mobile.html

```diff
@@ -1,126 +1,133 @@
 {% assign current = page.url | downcase | split: '/' %}
 
 <aside class="sidenav-mobile">
-  <a class="sliding-panel-close" href="#">
-    <img src="/assets/img/close.svg" alt="close">
-  </a>
+  <button class="usa-button-dismiss sliding-panel-close"> <span class="usa-sr-only">Dismiss</span></button>
   <nav>
```

Comment:
@diego-ruiz-rei - What's the rationale behind switching to the `button` element? Legitimately curious.